config/routes.rb

```diff
@@ -79,11 +82,10 @@
   delete '/openid_connect/authorize' => 'openid_connect/authorization#destroy',
          as: :openid_connect_deny
 
-  get '/openid_connect/certs' => 'openid_connect/certs#index'
-
-  post '/openid_connect/token' => 'openid_connect/token#create'
-
-  get '/openid_connect/userinfo' => 'openid_connect/user_info#show'
+  get '/openid_connect/certs' => 'openid_connect/certs#index', as: :old_openid_connect_certs
+  post '/openid_connect/token' => 'openid_connect/token#create', as: :old_openid_connect_token
+  get '/openid_connect/userinfo' => 'openid_connect/user_info#show',
+      as: :old_openid_connect_userinfo
```

Comment:
What is the purpose of making this change backwards compatible? Since we haven't launched yet, can't we tell any clients that might be using the old endpoints to use the new endpoints?